themes/finna2/templates/Helpers/holding-callnumber.phtml

```diff
@@ -23,5 +23,8 @@
       </ul>
     </div>
   <?php endif; ?>
+  <?php if ($this->wayfinder): ?>
+    <div class="wayfinder-placeholder" data-location='<?=$this->htmlSafeJsonEncode($this->wayfinder)?>'></div>
```

Comment:
Add a `js-` prefix to the placeholder class to indicate that it's used in JS (this practice is a recent one as well).

Reply:
And double quotes are fine with `htmlSafeJsonEncode`